src/Runtime/OMUnique.inc

```diff
@@ -322,6 +321,11 @@ void produceY(const OMTensor *inputTensor, OMTensor *indices, int64_t sliceAxis,
   }
 }
 
+static bool isEmptyTensor(OMTensor *tensor) {
+  return (tensor == NULL) || (omTensorGetRank(tensor) <= 0) ||
+         (omTensorGetShape(tensor)[0] == 0);
```

Comment:
By using nullptr for None, do we still need to check rank and shape?

Reply:
Removed the isEmptyTensor funciont, and fixed the code to check "pointer == NULL" directly. Thanks!